src/inputSources/mouseInput.js

```diff
@@ -108,11 +108,9 @@ function mouseDown (e) {
     type: eventType
   };
 
-  const event = external.$.Event(eventType, eventData);
+  const eventPropogated = triggerEvent(eventData.element, eventType, eventData);
 
-  external.$(eventData.element).trigger(event, eventData);
-
-  if (event.isImmediatePropagationStopped() === false) {
+  if (eventPropogated) {
```

Comment:
Something is wrong with the changes to mouseInput but I am not sure what. I guess it has to do with the stopImmediatePropogation or preventDefault changes.

If you create a length measurement, and then try to grab and move it, it will fail and instead create a new one. This is because mouseDownActivate is firing instead of mouseDown. Any ideas?

Reply:
I think it might be because some of the listeners use jQuery, and I only return the dispatchEvent value. I'll try changing the return value of `triggerEvent` to combine the jQuery and dispatchEvent.

Reply:
Fixed, I think.

I changed it so that it will not keep triggering regular events if any jQuery listeners call `stopImmediatePropagation()`, and so it returns the correct value (false if stopped/preventDefault)